packages/EasyCodingStandard/src/Console/Command/CheckCommand.php

```diff
@@ -66,6 +73,18 @@ protected function configure(): void
             InputOption::VALUE_NONE,
             'Hide error table. Useful e.g. for fast check of error count.'
         );
+        $this->addOption(
+            Option::OUTPUT_FORMAT_OPTION,
+            null,
+            InputOption::VALUE_OPTIONAL,
+            sprintf('Select output format: %s.', implode(', ', Option::OUTPUT_FORMATS)),
+            Option::TABLE_OUTPUT_FORMAT
+        );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->setOutputFormatter($this->container, $input);
     }
```

Comment:
Since Input + Output are services under EasyCodingStandardStyle, `initialize()` is not needed.

It's purporse it to hack around value objects in old non-service Symfony.

Reply:
I agree. I was experimenting, trying to find a way to make `--output-format` work.